modules/backup-and-restore/pages/cross-cluster-backup.adoc

```diff
@@ -12,78 +12,96 @@ Your cluster is offline during the restore process.
 * You have access to the TigerGraph Linux user account on your cluster.
 All commands must be run from the TigerGraph Linux user.
 * You have access to a data backup from another cluster.
+* [xref:_procedure_v3_9_1_and_below[3.9.1 and earlier]] You have the metadata backup that goes with the data backup.
 * You have xref:configurations.adoc[configured backup and restore] on the new cluster.
 
-NOTE: Since v3.9.2, it is no longer required to create a backup for a backups metadata or upload the metadata to a new cluster.
-A backup’s metadata is now stored as a part of a backup's files, but the previous method is still supported. *See xref:_prerequisites_v3_9_1_and_below[].
-
-Simply run:
-
-[source,console,]
---
-gadmin backup restore <backup_tag>
---
+== Procedure
 
-Just like restoring a backup in its original cluster.
+=== 1) Copy Data Backup to a New Cluster
 
-== Copy data backup to new cluster
+==== For Amazon S3 Backup:
 
-For an Amazon S3 backup, the restore flow will try to find a S3 object:
+If your backups are stored in Amazon S3 buckets, you can skip this step as long as you configure:
+[source,console]
+System.Backup.S3.BucketName
 
-[source, console]
---
-<backup_tag>_metadata
---
+For the new cluster to be the same as the previous cluster
 
-From the Bucket `System.Backup.S3.BucketName`.
+==== For Local Backup Storage:
 
 If your backups were stored locally, you need to make sure backup files from all nodes are copied to the new cluster.
-The restore flow will try to find a metadata file under the backup path:
+They do not necessarily need to be on the corresponding nodes as long as all partitions are in the new cluster at the specified paths you configured.
 
+* For example:
++
+.if you have configured the backup path to be:
+[source,console]
+/home/tigergraph/backups
++
+.For a 3-node cluster, you can put all backup files for backup:
 [source,console]
---
-/path/to/{System.Backup.Local.Path}/<backup_tag>/metadata
---
+weekly-<timestamp>
++
+.In the folder like this:
```

Comment:
```suggestion
.in the folder like this:
```
No capitalization, but a colon because we're reached the end of the sentence.

Reply:
updated with latest commits. Thank you.